draft-ietf-masque-h3-datagram.md

```diff
@@ -199,12 +199,11 @@ CAPSULE allows reliably sending request-related information end-to-end, even in
 the presence of HTTP intermediaries.
 
 CAPSULE is an HTTP/3 Frame (as opposed to a QUIC frame) which SHALL only be
-sent in client-initiated bidirectional streams. Intermediaries MUST forward all
-received CAPSULE frames in their unmodified entirety on the same stream where
-it would forward DATA frames. Intermediaries MUST NOT send any CAPSULE frames
-other than the ones it is forwarding. Intermediaries respect the order of
-CAPSULE frames: if an intermediary receives two CAPSULE frames in a given
-order, it MUST forward them in the same order.
+sent in client-initiated bidirectional streams. Intermediaries forward received
+CAPSULE frames in their unmodified entirety on the same stream where it would
+forward DATA frames. Each Capsule Type determines whether it is opaque or
+transparent to intermediaries: opaque capsules are forwarded unmodified while
+transparent ones can be parsed or added by intermediaries.
```

Comment:
Is 'parsed or added' what we want to say, or rather, 'transparent ones can be converted to semantically equivalent protocol features or extensions by an intermediary, as defined by the specific Capsule Type', or something like that.

Reply:
I'd rather leave this vague. If it's transparent, the intermediary is allowed to do anything to a capsule if it wants to. I tweaked the text to clarify the intent.